README.md

```diff
@@ -21,7 +21,7 @@ Perhaps you want people to be able to add labels to their own newly created issu
 
 ### Configuring Customized Responses
 
-You can use the [default responses](lib/defaults.js), but if you'd like to specify your own, create a file inside a `.github` folder, titled `teacherbot.yml` and include the following text:
+You can use the [default responses](https://github.com/teacher-bot/teacherbot/blob/master/index.js), but if you'd like to specify your own, create a file inside a `.github` folder, titled `teacherbot.yml` and include the following text:
```

Comment:
Yeah! We can even use [this fancy new feature](https://github.com/teacher-bot/teacherbot/blob/effc9e8938159f6838263a85fdd2cb3576cf463c/index.js#L6-L7)!

Reply:
@hectorsector I considered it, but it links to a specific tree and I didn't want to mess with updating it if this file changed :wink: